hl7/4.2/modules/ROOT/edits/hl7-connector-examples.adoc

```diff
@@ -0,0 +1,278 @@
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+  xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+  http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http
+http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/hl7
+http://www.mulesoft.org/schema/mule/hl7/current/mule-hl7.xsd
+http://www.mulesoft.org/schema/mule/ee/core
+http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+	<http:listener-config name="HTTP_Listener_config"
+  doc:name="HTTP Listener config" >
+		<http:listener-connection host="localhost" port="8081" />
+	</http:listener-config>
+	<hl7:config name="HL7_Extension_Config" doc:name="HL7 Extension Config" identKeys="true">
+		<hl7:schemas >
+			<hl7:schema value="/hl7/v2_5_1/ADT_A05.esl" />
+			<hl7:schema value="/hl7/v2_5_1/ADT_A01.esl" />
+		</hl7:schemas>
+	</hl7:config>
+	<flow name="hl7testFlow" >
+		<http:listener doc:name="Listener" config-ref="HTTP_Listener_config" path="/hl7"/>
+		<hl7:read doc:name="Read" config-ref="HL7_Extension_Config"/>
+		<ee:transform doc:name="Transform Message" >
+			<ee:message >
+				<ee:set-payload ><![CDATA[%dw 2.0
+output application/java
+---
+{
+	Delimiters: payload.Delimiters,
+	Id: payload.Id
+}]]></ee:set-payload>
+			</ee:message>
+		</ee:transform>
+		<hl7:write doc:name="Write" config-ref="HL7_Extension_Config"/>
+	</flow>
+</mule>
+----
+
+=== Set Your HL7 Identification in XML
+
+You can configure the Message Header (MSH) application and facility identification for you and your trading partner on the HL7 EDI connector configuration.
+
+The values you set are used when writing HL7 messages to supply the namespace ID, universal ID, and universal ID type, and are verified in receive messages. If you don't want to restrict incoming messages you can leave these blank, and set the values for outgoing messages on the write operation or the actual outgoing message. Values set on the write operation override the connector configuration, and values set directly on the message override both the connector configuration and any values set on the write operation.
+
+* Self identification parameters identify your side of the trading partner relationship.
++
+Self identification parameters:
++
+[source,text,linenums]
+----
+appNamespaceIdSelf="<value>"
+appUniversalIdSelf="<value>"
+appUniversalIdTypeSelf="<value>"
+----
++
+* Partner identification parameters identify your trading partner.
++
+Partner identification parameters:
++
+[source,text,linenums]
+----
+appNamespaceIdPartner="<value>"
+appUniversalIdPartner="<value>"
+appUniversalIdTypePartner="<value>"
+----
+
+=== Reading and Validating HL7 ER7 Messages
+
+To read an HL7 message:
+
+. Search the palette for *HL7 EDI* and drag it into a flow.
```

Comment:
Is the audience searching by scanning for a label (bold) or using a search tool (lowercase and mono)?